public/app/plugins/datasource/influxdb/influx_series.js

```diff
@@ -81,15 +81,15 @@ function (_, TableModel) {
     _.each(this.series, function (series) {
       var titleCol = null;
       var timeCol = null;
-      var tagsCol = null;
+      var tagsCol = [];
       var textCol = null;
 
       _.each(series.columns, function(column, index) {
         if (column === 'time') { timeCol = index; return; }
         if (column === 'sequence_number') { return; }
         if (!titleCol) { titleCol = index; }
         if (column === self.annotation.titleColumn) { titleCol = index; return; }
-        if (column === self.annotation.tagsColumn) { tagsCol = index; return; }
+        if (self.annotation.tagsColumn.includes(column)) { tagsCol.push(index); return; }
```

Comment:
includes seems like a very imprecise way to check here, tagsColumn should be split before so a property exact match can be made